globals.go

```diff
@@ -621,14 +736,14 @@ func (p HostsPermission) Value() int {
 	return int(p)
 }
 
-var projectPermissions = []string{
+var ProjectPermissions = []string{
 	PermissionProjectSettings,
 	PermissionTasks,
 	PermissionPatches,
 	PermissionLogs,
 }
 
-var distroPermissions = []string{
+var DistroPermissions = []string{
 	PermissionDistroSettings,
 	PermissionHosts,
 }
```

Comment:
there should probably be a note here that if you update this you have to also update the route, or find some way to organize the route code to be able to handle this.

Reply:
I believe the route code already handles this

Reply:
the route code will automatically handle the addition or removal of new permissions